sdk/attestation/attestation/test/public/PolicyManagementGetSetTests.spec.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert, use as chaiUse } from "chai";
+import chaiPromises from "chai-as-promised";
+chaiUse(chaiPromises);
+
+import { isPlaybackMode, Recorder } from "@azure/test-utils-recorder";
+
+import { createRecordedClient, createRecorder } from "../utils/recordedClient";
+import { AttestationClient } from "../../src";
+import { verifyAttestationToken } from "../utils/helpers";
+
+describe("PolicyManagementTests ", function() {
+  let recorder: Recorder;
+
+  beforeEach(function() {
+    // eslint-disable-next-line no-invalid-this
+    recorder = createRecorder(this);
+  });
+
+  afterEach(async function() {
+    await recorder.stop();
+  });
+
+  it("#GetPolicyManagementCertificatesAad", async () => {
+    let client: AttestationClient;
+    client = createRecordedClient("AAD");
+
+    const policyResult = await client.policyCertificates.get();
+    const result = policyResult.token;
+    assert(result);
+    if (result && !isPlaybackMode()) {
+      const tokenResult = await verifyAttestationToken(result, client);
+      assert.isDefined(tokenResult);
+      if (tokenResult) {
+        const tokenKeys = tokenResult["x-ms-policy-certificates"];
+        assert.equal(tokenKeys.keys.length, 0);
+      }
+    }
+  });
+
+  it("#GetPolicyShared", async () => {
+    let client: AttestationClient;
+    client = createRecordedClient("Shared");
+    const policyResult = await client.policyCertificates.get();
+
+    const result = policyResult.token;
+    assert(result);
```

Comment:
[NIT]

Recommend adding the message for the failure case - `assert(result, /*messsage to show on error*/);`.